src/utils/media-utils.js

```diff
@@ -334,3 +334,8 @@ export function injectCustomShaderChunks(obj) {
 export function getPromotionTokenForFile(fileId) {
   return window.APP.store.state.uploadPromotionTokens.find(upload => upload.fileId === fileId);
 }
+
+const hubsSceneRegex = /https?:\/\/(hubs.local(:\d+)?|(smoke-)?hubs.mozilla.com)\/scenes\/(\w+)\/?\S*/;
+const hubsRoomRegex = /https?:\/\/(hubs.local(:\d+)?|(smoke-)?hubs.mozilla.com)\/(\w+)\/?\S*/;
```

Comment:
Seems like we should restrict this a bit. Our hub ids are of a known length, so enforcing that might help cut down false positives a bit.

Reply:
unfortunately we have some legacy ones that are longer :(

i'll see if i can tighten this up at all though (on the next branch, which moves these utilities somewhere else)

Reply:
i'm a bit nervous about this being future proofed if i set the length. there's a lot of coupling with the reticulum routing logic. main issue here is going to be false positives, but the only side effect is that it will say "visit room" instead of "open url" -- so nbd for now.